content/2022/05-18-new-endpoints-for-user-avatar.md

```diff
@@ -0,0 +1,37 @@
+---
+applied_at: "2022-05-18"
+applies_to: 
+- api
+is_impactful: true
+is_new_feature: true
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# New Endpoints to Manage User Avatars
+
+The User Avatar API now supports adding, updating, and deleting user avatars.
+
+To create or update a user avatar, call the [`POST /user-avatar`][2] endpoint. Remember to include the path to the file you want to upload.
```

Comment:
```suggestion
To create or update a user avatar, call the [`POST /user-avatar`][2] endpoint. Remember to include the path to the file you want to upload. If an avatar already exists, it will be replaced with the new photo uploaded.
```